src/Worker.php

```diff
@@ -80,9 +80,11 @@ private function tick($queue)
         }
 
         try {
+            $this->event->acknowledge($message);
```

Comment:
We should first define what `acknowledge` means. Does it mean that yeah we got the message but haven't done anything with it yet or we are ready to start executing the job associated with this message?